libs/portal-integration-angular/src/lib/core/components/data-view-controls/data-view-controls.component.html

```diff
@@ -66,6 +71,7 @@
         styleClass="data-view-control-border min-w-min w-7rem"
         [options]="sortingOptions"
         (onChange)="selectSorting($event)"
+        [ariaLabel]="('OCX_LIST_GRID_SORT.DROPDOWN.ARIA_LABEL' | translate)"
```

Comment:
[attr.aria-label] ?

Reply:
p-dropdown has `ariaLabel` prop which should be used in this case